layouts/default.vue

```diff
@@ -139,69 +137,69 @@
+
+          <nuxt-link v-if="loggedIn" id="menu-option-chat-sm" class="text-white mr-3" to="/chats">
             <div class="notifwrapper">
-              <v-icon name="bell" scale="2" class="" />
-              <b-badge v-if="notificationCount" variant="danger" class="notifbadgesm">
-                {{ notificationCount }}
+              <v-icon name="comments" scale="2" /><br>
+              <b-badge v-if="chatCount" variant="danger" class="chatbadge">
+                {{ chatCount }}
               </b-badge>
             </div>
-          </template>
-          <b-dropdown-item class="text-right">
-            <b-btn variant="white" size="sm" @click="markAllRead">
-              Mark all read
-            </b-btn>
-          </b-dropdown-item>
-          <b-dropdown-divider />
-          <b-dropdown-item v-for="notification in notifications" :key="'notification-' + notification.id" class="p-0 notpad">
-            <Notification :notification="notification" class="p-0" @showModal="showAboutMe" />
-          </b-dropdown-item>
-          <infinite-loading :distance="distance" @infinite="loadMoreNotifications">
-            <span slot="no-results" />
-            <span slot="no-more" />
-            <span slot="spinner">
-              <b-img-lazy src="~/static/loader.gif" alt="Loading" />
-            </span>
-          </infinite-loading>
-        </b-dropdown>
-
-        <nuxt-link v-if="loggedIn" id="menu-option-chat-sm" class="text-white mr-3" to="/chats">
-          <div class="notifwrapper">
-            <v-icon name="comments" scale="2" /><br>
-            <b-badge v-if="chatCount" variant="danger" class="chatbadge">
-              {{ chatCount }}
-            </b-badge>
-          </div>
-        </nuxt-link>
-      </client-only>
-
-      <b-navbar-nav>
-        <client-only>
-          <b-nav-item v-if="!loggedIn">
-            <div class="btn btn-white" @click="requestLogin">
-              Sign in
-            </div>
-          </b-nav-item>
+          </nuxt-link>
         </client-only>
-      </b-navbar-nav>
 
-      <b-navbar-nav class="">
-        <b-btn v-if="loggedIn" v-b-toggle.nav_collapse_mobile class="toggler white">
-          <v-icon name="bars" class="mb-1" scale="1.5" />
-        </b-btn>
-        <!--        <b-navbar-toggle v-if="loggedIn" target="nav_collapse_mobile" class="navbar-dark" />-->
-      </b-navbar-nav>
+        <b-navbar-nav>
+          <client-only>
+            <b-nav-item v-if="!loggedIn">
+              <div class="btn btn-white" @click="requestLogin">
+                Sign in
+              </div>
+            </b-nav-item>
+          </client-only>
+        </b-navbar-nav>
 
+        <b-navbar-nav class="">
+          <b-btn v-if="loggedIn" v-b-toggle.nav_collapse_mobile class="toggler white">
+            <v-icon name="bars" class="mb-1" scale="1.5" />
+          </b-btn>
+        <!--        <b-navbar-toggle v-if="loggedIn" target="nav_collapse_mobile" class="navbar-dark" />-->
```

Comment:
Any point in this code if we're commenting it out? Looks like legacy but id be tempted to remove it, its just noise

Reply:
@edwh - Can this be removed or is it a reminder for something?

Reply:
It can be removed.

Reply:
Done. :-)